tests/multigrid_evolve_test.cpp

```diff
@@ -395,6 +407,19 @@ int main(int argc, char **argv)
         if (inv_multigrid) mg_param.invert_param->mu = inv_param.mu;
       }
 
+      // as needed to bake in mu
+      if (dslash_type == QUDA_TWISTED_CLOVER_DSLASH) {
```

Comment:
Why is the clover field only being recomputed for twisted-clover here and not regular clover?  Since this loop will be modifying the mass and not just the twist, I think we need to recompute the clover field.

Reply:
Good catch, fixed.